climada/util/coordinates.py

```diff
@@ -76,6 +78,256 @@
 MAX_DEM_TILES_DOWN = 300
 """Maximum DEM tiles to dowload"""
 
+######### copied over from interpolation module #####################
+DIST_DEF = ['approx', 'haversine']
+"""Distances"""
+
+METHOD = ['NN']
+"""Interpolation methods"""
+
+THRESHOLD = 100
+"""Distance threshold in km. Nearest neighbors with greater distances are
+not considered."""
+
+
+@jit(nopython=True, parallel=True)
+def _dist_sqr_approx(lats1, lons1, cos_lats1, lats2, lons2):
+    """Compute squared equirectangular approximation distance. Values need
+    to be sqrt and multiplicated by ONE_LAT_KM to obtain distance in km."""
+    d_lon = lons1 - lons2
+    d_lat = lats1 - lats2
+    return d_lon * d_lon * cos_lats1 * cos_lats1 + d_lat * d_lat
+
+def interpol_index(centroids, coordinates, method=METHOD[0],
+                   distance=DIST_DEF[1], threshold=THRESHOLD):
+    """Returns for each coordinate the centroids indexes used for
```

Comment:
Could you please update the docstrings to numpy format? Otherwise this will clash with the current update of the docstrings.

Reply:
yep. 946d4f11